### DISCUSSION TITLE
Enable 2FA

### DISCUSSION THREAD

**Discussion by zappatero** [directus/directus#5666] [Archive]
How needs the 2FA to be configured?
I have an admin user in the admin group. The admin group is configured to force 2FA.
However when logging in as admin, no 2FA prompt appears.
On the admin user in the backend there is only an empty 2FA field shown.

Thx for help or docs

**Comment by rijkvanzanten** [comment]
> However when logging in as admin, no 2FA prompt appears.

This seems to be a bug! I've opened https://github.com/directus/directus/issues/5682 to track this 👍🏻 

Thanks for bringing this up 🙂

**Reply by zappatero** [reply]
Thanks!

![image](https://user-images.githubusercontent.com/42898262/118322997-465c1780-b500-11eb-97ac-4dc7df987a0c.png)

Shouldn't this be a checkbox to activate the 2FA ?

**Reply by rijkvanzanten** [reply]
Yes it should, and now it is 👍🏻
